plugins/modules/mount.py

```diff
@@ -68,6 +68,13 @@
       - If C(unmounted), the device will be unmounted without changing I(fstab).
       - C(present) only specifies that the device is to be configured in
         I(fstab) and does not trigger or require a mount.
+      - C(ephemeral) only specifies that the device is to be mounted, without changing
+        I(fstab). If it is already mounted, a remount will be triggered.
+        This will always return changed=True. If the mount point I(path)
+        has already a device mounted on, and its source is different than I(src),
+        the module will fail to avoid unexpected unmount or mount point override.
+        If the mount point is not present, the mount point will be created.
+        The I(fstab) is completely ignored. This option is added in version 1.4.0.
```

Comment:
Oops, stupid me did not commit the change. Sorry for that. I did as I said in [my previous comment](https://github.com/ansible-collections/ansible.posix/pull/267#issuecomment-1145675009):

> I amended previous commit to avoid adding more commits, while keeping track of the co-authored change.